src/redist/targets/MacEntitlements/AddMacEntitlements.targets

```diff
@@ -0,0 +1,7 @@
+<Project>
+    <!-- Entitlements are needed but not automatically added for macOS. See https://github.com/dotnet/runtime/issues/113707 -->
+    <!-- This generates an ad-hoc signature that will later be resigned, but keeps the entitlements. -->
+    <Target Name="AddEntitlements" AfterTargets="GenerateLayout">
+        <Exec Command="codesign -s - -f --entitlements $(MSBuildThisFileDirectory)entitlements.plist $(ArtifactsDir)layout/dotnet-core-uninstall/dotnet-core-uninstall" />
```

Comment:
akoeplinger's version also had the `-` by itself, so I'm assuming it's correct, but it looked weird to me. What does this mean exactly?

Reply:
It refers to an empty certificate name. This is an ad-hoc signing, and it will be signed again, so I think it wouldn't make much difference